### PR TITLE
Clean up CHANGELOG for v0.6.28 release

### PR DESCRIPTION
**Pull request by cdeszaq**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,11 +11,58 @@ Current
...
+
+### Added:
+
 - [Customize Logging Format in RequestLog](https://github.com/yahoo/fili/pull/81)
 
-- [Populate Dimension Rows from an AVRO file](https://github.com/yahoo/fili/pull/86)
-    * Added `AvroDimensionRowParser` class that parses an AVRO data file (.avro) after validating the AVRO schema (.avsc).
-    * Added a functional Interface `DimensionFieldMapper` that maps field name based on the user implementation.
+- [Can Populate Dimension Rows from an AVRO file](https://github.com/yahoo/fili/pull/86)
+    * Added `AvroDimensionRowParser` that parses an AVRO data file int `DimensionRow`s after validating the AVRO schema.
```
Comment (archolewa) [inline CHANGELOG.md]: `int` should be `into` in `an AVRO data file int`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+[![Download](https://api.bintray.com/packages/yahoo/maven/fili/images/download.svg)](https://bintray.com/yahoo/maven/fili/_latestVersion) [![Gitter](https://img.shields.io/gitter/room/yahoo/fili.svg?maxAge=2592000)](https://gitter.im/yahoo/fili) [![Travis](https://img.shields.io/travis/yahoo/fili/master.svg?maxAge=2592000)](https://travis-ci.org/yahoo/fili/builds/) [![Codacy grade](https://img.shields.io/codacy/grade/91fa6c38f25d4ea0ae3569ee70a33e38.svg?maxAge=2592000)](https://www.codacy.com/app/Fili/fili/dashboard) [![Codacy coverage](https://img.shields.io/codacy/coverage/91fa6c38f25d4ea0ae3569ee70a33e38.svg?label=test coverage&maxAge=2592000)](https://www.codacy.com/app/Fili/fili/dashboard) [![VersionEye](https://img.shields.io/versioneye/d/user/projects/57eb1529bd6fa600512e4af1.svg?maxAge=2592000)](https://www.versioneye.com/user/projects/57eb1529bd6fa600512e4af1)
```
Comment (archolewa) [inline README.md]: Errm, what changed here?

**Comment by archolewa** [comment]
A small thing or two, 👍 once they've been addressed.

File: CHANGELOG.md (hunk 1 above)
Comment (cdeszaq) [inline CHANGELOG.md]: Good catch, will fix.

File: README.md (hunk 2 above)
Comment (cdeszaq) [inline README.md]: A couple of random whitespaces got removed to make the link formats consistent.

**Comment by michael-mclawhorn** [comment]
👍

**Pull request merged**